tests/test_add_parent_mass.py

```diff
@@ -17,9 +17,24 @@ def test_add_parent_mass():
     spectrum = add_parent_mass(spectrum_in)
 
     assert numpy.abs(spectrum.get("parent_mass") - 445.0) < .01, "Expected parent mass of about 445.0."
+    assert isinstance(spectrum.get("parent_mass"), float), "Expected parent mass to be float."
 
 
 def test_add_parent_mass_no_pepmass():
+    """Test if correct expection is returned."""
+    mz = numpy.array([], dtype='float')
+    intensities = numpy.array([], dtype='float')
+    metadata = {"charge": -1}
+    spectrum_in = Spectrum(mz=mz,
+                           intensities=intensities,
+                           metadata=metadata)
+
+    spectrum = add_parent_mass(spectrum_in)
+
+    assert spectrum.get("parent_mass") is None, "Expected no parent mass"
```

Comment:
Also assert if print happened by using https://docs.pytest.org/en/stable/capture.html
```suggestion
def test_add_parent_mass_no_pepmass(capsys):
    """Test if correct expection is returned."""
    mz = numpy.array([], dtype='float')
    intensities = numpy.array([], dtype='float')
    metadata = {"charge": -1}
    spectrum_in = Spectrum(mz=mz,
                           intensities=intensities,
                           metadata=metadata)

    spectrum = add_parent_mass(spectrum_in)

    assert spectrum.get("parent_mass") is None, "Expected no parent mass"
    assert "Not sufficient spectrum metadata to derive parent mass." in capsys.readouterr().out
```

Reply:
Nice, thanks! Didn't know how to handle that before, so glad to see how that works.

Reply:
Just learned it myself, needed it for howfairis last week. New entry for @felipeZ [cookbook](https://blog.esciencecenter.nl/a-quick-and-dirty-pytest-cookbook-58051a2980c3)?